lib/client.js

```diff
@@ -7,12 +7,16 @@ const EventEmitter = require('events');
 const Resource = require('./resource');
 const Metrics = require('@metrics/client');
 const schemas = require('@podium/schemas');
+const abslog = require('abslog');
 const assert = require('assert');
 const Cache = require('ttl-mem-cache');
 const utils = require('./utils');
 const http = require('http');
 const joi = require('joi');
 
+// Workaround for https://github.com/podium-lib/issues/issues/8
+EventEmitter.defaultMaxListeners = 30;
```

Comment:
Would it make any difference to set this on the child vs the parent? ie. if you were to set it on `PodletClient` instead of `EventEmitter` wouldn't that scope this change to just instances of that class?